extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -67,6 +67,11 @@
 	'communitypage-noofviews' => '$1 views',
 	'communitypage-full-list' => 'View full list',
 	'communitypage-view-more' => 'View more',
+	'communitypage-policy-heading' => 'Community Policy',
+	'communitypage-policy-text' => 'This community has a set of policies that should be followed by all contributors.',
+	'communitypage-policy-edit' => 'Edit',
+	'communitypage-policy-view' => 'View policy »',
+	'communitypage-policy-pagetitle' => 'Community guidelines',
```

Comment:
Can we decide on naming and keep sufixes consistent with other modules? https://github.com/Wikia/app/pull/10522/files
They both seem to have same purpose and format.